datastore/datastore_test.go

```diff
@@ -50,6 +50,8 @@ type KeyValue struct {
 	Value string
 }
 
+var keyValueCache Cache // KeyValue cache.
+
```

Comment:
I still don't think it should be global in testing though; unit tests should not have the capacity to pollute other unit tests through a global. Unit tests should be self contained.

Reply:
I'm not sure how to dispense with the global, without making the cache implementation a property of KeyValue.
Conceptually, I want to be able to switch out the GetCache implementation. A closure would not seem to help here.